core-bundle/tests/Contao/StringUtilTest.php

```diff
@@ -218,12 +218,6 @@ public function parseSimpleTokensProvider(): \Generator
             '{if token=="foo"}',
         ];
 
-        yield 'Test nested if-tag with " in value (match)' => [
```

Comment:
That's the only thing that's technically a BC break. However, I don't know why we ever supported that because it's clearly invalid syntax. I hope that's okay. /cc @ausi

Reply:
I’m OK with breaking that.